### PR TITLE
[seal] Updated SEAL to version 3.7.0

### PR DESCRIPTION
**Pull request by WeiDaiWD**
**Describe the pull request**

- #### What does your PR fix?  

Updated `ports/seal` to version 3.7.0.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  

x64-windows/x86-windows are not supported as the library only supports static linkage in Windows.
No change to the CI baseline.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  

Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  

Yes.

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/seal/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "seal",
-  "version-semver": "3.6.6",
+  "version-semver": "3.7.0",
   "port-version": 1,
```
Comment (PhoebeHui) [inline ports/seal/vcpkg.json:4]:
```suggestion
```

**Comment by PhoebeHui** [comment]
@WeiDaiWD, thanks for the PR!

Please ignore the CI failures, it's a baseline issue, we're taking look at it now.

**Comment by WeiDaiWD** [comment]
Will you re-run the pipelines when the issue is fixed?

**Comment by PhoebeHui** [comment]
apsi with x64-windows-static, x64-windows-static-md, x64-linux and x64-osx failed in CI testsing, could you please take a look?

```
CMake Error at C:/a/1/s/scripts/buildsystems/vcpkg.cmake:786 (_find_package):
  Could not find a configuration file for package "SEAL" that is compatible
  with requested version "3.6".

  The following configuration files were considered but not accepted:
    D:/installed/x64-windows-static/share/SEAL/SEALConfig.cmake, version: 3.7.0
Call Stack (most recent call first):
  CMakeLists.txt:111 (find_package)
```

**Comment by WeiDaiWD** [comment]
> _This is a new experimental fast check for PR issues. Please let us know if this bot is helpful!_
> 
> After committing all other changes, the version database must be updated

This is resolved.

**Comment by WeiDaiWD** [comment]
> apsi with x64-windows-static, x64-windows-static-md, x64-linux and x64-osx failed in CI testsing, could you please take a look?
> 
> ```
> CMake Error at C:/a/1/s/scripts/buildsystems/vcpkg.cmake:786 (_find_package):
>   Could not find a configuration file for package "SEAL" that is compatible
>   with requested version "3.6".
> 
>   The following configuration files were considered but not accepted:
>     D:/installed/x64-windows-static/share/SEAL/SEALConfig.cmake, version: 3.7.0
> Call Stack (most recent call first):
>   CMakeLists.txt:111 (find_package)
> ```

I updated APSI too.

**Comment by PhoebeHui** [comment]
@WeiDaiWD, thanks for your updates! I tested the all features with x64-windows locally, it failed with following failures, could you address the issue?


```
-- Performing post-build validation

The following dlls were found in /lib or /debug/lib. Please move them to /bin or /debug/bin, respectively.

    F:/vcpkg/20166/vcpkg/packages/hexl_x64-windows/lib/hexl.dll


The following dlls were found in /lib or /debug/lib. Please move them to /bin or /debug/bin, respectively.

    F:/vcpkg/20166/vcpkg/packages/hexl_x64-windows/debug/lib/hexl_debug.dll

Import libs were not present in F:\vcpkg\20166\vcpkg\packages\hexl_x64-windows\debug/lib
If this is intended, add the following line in the portfile:
    SET(VCPKG_POLICY_DLLS_WITHOUT_LIBS enabled)
Import libs were not present in F:\vcpkg\20166\vcpkg\packages\hexl_x64-windows\lib
If this is intended, add the following line in the portfile:
    SET(VCPKG_POLICY_DLLS_WITHOUT_LIBS enabled)
The following DLLs have no exports:

    F:/vcpkg/20166/vcpkg/packages/hexl_x64-windows/debug/bin/hexl_debug.dll
    F:/vcpkg/20166/vcpkg/packages/hexl_x64-windows/bin/hexl.dll

DLLs without any exports are likely a bug in the build script.
If this is intended, add the following line in the portfile:
    SET(VCPKG_POLICY_DLLS_WITHOUT_EXPORTS enabled)
Found 5 error(s). Please correct the portfile:
    F:\vcpkg\20166\vcpkg\ports\hexl\portfile.cmake
-- Performing post-build validation done
Error: Building package hexl:x64-windows failed with: POST_BUILD_CHECKS_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: hexl:x64-windows
  Vcpkg version: 2021-09-10-2059ef11aa6067e6f59b0d939c5d17e3c5c47d3e

Additionally, attach any relevant sections from the log files above.
```

**Review by PhoebeHui — APPROVED** [review]

**Pull request merged**